features/support/env.rb

```diff
@@ -17,6 +17,7 @@
 class Cucumber::Rails::World
   require Rails.root.join('spec', 'support', 'common_helper_methods.rb')
   require Rails.root.join('spec', 'support', 'scenario_progress_formatter.rb')
+  require Rails.root.join('spec', 'support', 'shared_helper.rb')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.